app/views/songs/index.html.erb

```diff
@@ -39,7 +39,9 @@
       serverSide: true,
       responsive: true,
       lengthChange: false,
-      dom: '',
+      pageLength: 25,
```

Comment:
I'm also going to change when queries are submitted to the server. Right now it's firing off way too many (on each keystroke) and things are slower than I'd like. I'm making it so that it'll only search 1 second after the last keystroke (like Google does). This works really well.